checksums.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Please unstage this file from the commit. This is created by `puppet module build` for the Forge and is not meant to be checked into the repo.